lib/basedriver/driver.js

```diff
@@ -140,6 +171,14 @@ class BaseDriver extends MobileJsonWireProtocol {
   // and ensuring that we execute commands one at a time. This method is called
   // by MJSONWP's express router.
   async executeCommand (cmd, ...args) {
+    // get start time for this command, and log in special cases
+    let startTime = Date.now();
```

Comment:
I wasn't sure if all we care about is elapsed time. If not, then we'd have a mix of elapsed times and absolute times in the event log, which seemed odd. OTOH if we just use absolute times everywhere, then a client can figure out what it cares about and determine elapsed time between any two events it cares about. So it seemed more flexible, if a bit more indirect from a client perspective.

Again, I'm willing to be convinced, this just seemed cleanest somehow.